lib/apipie_bindings/api.rb

```diff
@@ -90,6 +90,9 @@ def initialize(config, options={})
 
       @credentials = config[:credentials] if config[:credentials] && config[:credentials].respond_to?(:to_params)
 
+      if (config[:timeout] && config[:timeout].to_i < 0)
+        config[:timeout] = (RestClient.version < '2.0.0') ? -1 : nil
```

Comment:
This should be `< 1.7.0` rather than 2.0.0, it's been deprecated longer than this: https://github.com/rest-client/rest-client/blob/v1.7.0/history.md